md_link_check_config.json

```diff
@@ -0,0 +1,15 @@
+{
+    "ignorePatterns": [
+      {
+        "pattern": "^https://github.com"
```

Comment:
we have to ignore github.com addresses or otherwise we'll get "too many requests" errors.